### ISSUE TITLE
[RPC] eth_gasPrice with no arguments error

### ISSUE THREAD

**Issue by hexoscott** [0xPolygon/cdk-erigon#1397]
In hermeznetwork/cdk-erigon:v2.0.0 we get the following error when calling eth_gasPrice:
```
{
    "jsonrpc": "2.0",
    "method": "eth_gasPrice",
    "params": [],
    "id": 1
}
```

`json: cannot unmarshal object into Go struct field ErrorObject.Error.data of type *types.ArgBytes`

**Comment by MorettiGeorgiev** [comment]
Can't reproduce this one, tried with the hermeznetwork/cdk-erigon:v2.0.0 image and the gas price is returned without error

**Comment by hexoscott** [comment]
Fixed by #1410
